pipelines/skylab/optimus/scripts/export_terra.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+declare -r namespace=${WORKSPACE_NAMESPACE}
+declare -r name=${WORKSPACE_NAME}
+
+python workspace_controller.py get_participant_table \
```

Comment:
given nikelle's pr to upgrade python 2 to python 3 is it worth specifying python3 here? or adding a task to her pr to upgrade this as well?

Reply:
good looks, probably just easier to specify python3 here